src/mds/Server.cc

```diff
@@ -5689,8 +5689,34 @@ void Server::handle_set_vxattr(MDRequestRef& mdr, CInode *cur)
       return;
     }
 
-    if (!xlock_policylock(mdr, cur, false, true))
-      return;
+    /* Verify it's not already a subvolume with lighter weight
+     * rdlock.
+     */
+    {
```

Comment:
any reason this is under a separate section?

Reply:
Scope the variables lifetime.

Reply:
ah ok!